src/test/java/bio/terra/service/filedata/azure/blobstore/AzureBlobStorePdaoTest.java

```diff
@@ -160,10 +161,10 @@ public void setUp() {
     blobCrl = mock(BlobCrl.class);
     doReturn(targetBlobContainerFactory).when(dao).getTargetDataClientFactory(any(), any(), any());
```

Comment:
That's fair enough, I myself am not grokking it fully either :)

I don't think this is a critical improvement at this stage, so I'll resolve this convo.  I think it may be easier to grok that last mile of incremental improvement once this PR is in.